docs/kubernetes/windows.md

```diff
@@ -1,446 +1,358 @@
-
-6. Type `kubectl get svc -w` to watch the addition of the external IP address that will take about 2-5 minutes.  Once there, you can take the external IP and view in your web browser.
-
-## Example using Azure Files and Azure Disks
-### Create Azure File workload
-This example is modified after https://github.com/andyzhangx/Demo/tree/master/windows/azurefile/rs3
-
-#### 1. Create an azure file storage class
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/storageclass-azurefile.yaml```
-
-#### make sure storageclass is created successfully
-```
-kubectl get storageclass/azurefile -o wide
+```powershell
+dir ~\.ssh\id_rsa.pub
+dir : Cannot find path 'C:\Users\patrick\.ssh\id_rsa.pub' because it does not exist.
 ```
 
-#### 2. Create a pvc for azure file
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/pvc-azurefile.yaml```
-
-#### make sure pvc is created successfully
-```
-kubectl get pvc/pvc-azurefile -o wide
+If it does not exist, then run `ssh-keygen.exe`. Use the default file, and enter a passphrase if you wish to protect it. Be sure not to use a SSH key with blank passphrase in production.
+
+```powershell
+PS C:\Users\patrick\acs-engine> ssh-keygen.exe
+Generating public/private rsa key pair.
+Enter file in which to save the key (C:\Users\patrick/.ssh/id_rsa):
+Created directory 'C:\Users\patrick/.ssh'.
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in C:\Users\patrick/.ssh/id_rsa.
+Your public key has been saved in C:\Users\patrick/.ssh/id_rsa.pub.
+The key fingerprint is:
+SHA256:... patrick@plang-g1
+The key's randomart image is:
++---[RSA 2048]----+
+...
++----[SHA256]-----+
 ```
 
-#### 3. Create a pod with azure file pvc
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/iis-azurefile.yaml```
+### Create a Resource Group and Service Principal
 
-#### watch the status of pod until its `STATUS` is `Running`
-```
-watch kubectl get po/iis-azurefile -o wide
-```
+Now that we have the Azure CLI configured and a SSH key generated, it's time to create a resource group to hold the deployment.
 
-#### 4. Enter the pod container to validate
-```
-kubectl exec -it iis-azurefile -- cmd
-```
+`az group create --location <location> --name <name>` will create a group for you. Be sure to use a unique name for each cluster. If you need a list of available locations, run `az account list-locations -o table`.
 
+```powershell
+PS C:\Users\patrick\acs-engine> az group create --location westus2 --name k8s-win1
+{
+  "id": "/subscriptions/df392461-0000-1111-2222-cd3aa2d911a6/resourceGroups/k8s-win1",
+  "location": "westus2",
+  "managedBy": null,
+  "name": "k8s-win1",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null
+}
 ```

```

Comment:
Add second code sample for ```bash``` (even though they are the same commands, want to make sure someone doesn't skim over something they feel is only for those performing this deployment from a Windows platform